http/server.ts

```diff
@@ -599,6 +697,8 @@ export async function serve(
  * @param addr The address to listen on.
  * @param handler The handler for individual HTTP requests.
  * @param options Optional serve options.
+ *
+ * @deprecated Use `serve` instead.
```

Comment:
Can you put these all the way at the top of the JSDOC?

Reply:
Sure. Done!